src/components/Sum.jsx

```diff
@@ -1 +1,19 @@
 // Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    {'Sum of '}
+    {a}
+    {' and '}
+    {b}
+    {' is '}
+    {a + b}
```

Comment:
You can use this {`Sum of ${a} and ${b} is ${a + b}`}

Reply:
Yep. It would be more readable